packages/pcre/pcre.7.1.6/files/pcre.install

```diff
@@ -0,0 +1,6 @@
+etc: [
+  "setup.ml"
+  "setup.data"
+  "setup.log"
+  "_oasis_remove_.ml"
+]
```

Comment:
@Chris00 It seems that these lines were added by the latest oasis2opam.  I'm not sure all of these need to be installed.  Could you please let me know what to do about it?